pkg/arvo/sys/vane/clay.hoon

```diff
@@ -1445,6 +1445,32 @@
       ==
     ==
   ::
+  :: Attach label to aeon
+  ::
+  ++  label
+    |=  [bel=@tas aey=(unit aeon)]
+    ^+  ..park
+    =/  yon  ?~(aey let.dom u.aey)
+    =/  yen  (~(get by lab.dom) bel)  :: existing aeon?
+    ::  no existing aeon is bound to this label
+    ::
+    ?~  yen
+      =.  lab.dom  (~(put by lab.dom) bel yon)
+      ..park
+    :: an aeon is bound to this label, 
```

Comment:
By convention (at least in the kernel?) comments in hoon start with two spaces, so they're aligned with runes.